stdlib/nghttp2_jll/Project.toml

```
@@ -1,6 +1,6 @@
 name = "nghttp2_jll"
 uuid = "8e850ede-7688-5339-a07c-302acd2aaf8d"
-version = "1.41.0+1"
+version = "1.47.0+1"
```

The new version is [`1.47.0+0`](https://github.com/JuliaBinaryWrappers/nghttp2_jll.jl/releases/tag/nghttp2-v1.47.0%2B0) 
```suggestion
version = "1.47.0+0"
```
After this, you should be able to refresh the checksums

---

Doh.